### PR TITLE
Fixes for Objects examples

### PR DESCRIPTION
**Pull request by gmauro**

### CODE REVIEW

**Comment by simonbray** [comment]
@gmauro are you finished here?

**Comment by gmauro** [comment]
One of them is still failing but it requires a bit of work to fix it. I will open a different PR.
So probably, yes. I finished.

**Comment by simonbray** [comment]
Could you fix the python linting? Then I guess it can be merged.

File: docs/examples/objects/small.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import tempfile
 
 from bioblend.galaxy.objects import GalaxyInstance
-from .common import get_one
+from common import get_one
```
Comment (nsoranzo) [inline docs/examples/objects/small.py:6]:
```suggestion
from common import get_one  # noqa:I100,I201
```

File: docs/examples/objects/w2_bacterial_reseq.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import sys
 
 from bioblend.galaxy.objects import GalaxyInstance
-from .common import get_one
+from common import get_one
```
Comment (nsoranzo) [inline docs/examples/objects/w2_bacterial_reseq.py:5]:
```suggestion
from common import get_one  # noqa:I100,I201
```

File: docs/examples/objects/w3_bacterial_denovo.py [hunk 3]
```diff
@@ -1,8 +1,9 @@
+import json
 import os
 import sys
 
 from bioblend.galaxy.objects import GalaxyInstance
-from .common import get_one
+from common import get_one
```
Comment (nsoranzo) [inline docs/examples/objects/w3_bacterial_denovo.py:6]:
```suggestion
from common import get_one  # noqa:I100,I201
```

File: docs/examples/objects/w5_metagenomics.py [hunk 4]
```diff
@@ -1,8 +1,9 @@
+import json
 import os
 import sys
 
 from bioblend.galaxy.objects import GalaxyInstance
-from .common import get_one
+from common import get_one
```
Comment (nsoranzo) [inline docs/examples/objects/w5_metagenomics.py:6]:
```suggestion
from common import get_one  # noqa:I100,I201
```

**Pull request merged**